src/components/OrderDetails.js

```diff
@@ -1,14 +1,62 @@
 import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import actions from '../action_creators';
+import _ from 'underscore';
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
```

Comment:
A couple friends were in town this weekend so didn't get a chance to work on it.. will make some progress tonight.